### ISSUE TITLE
Invalid long_short value of exit_trade() function

### ISSUE THREAD

**Issue by vinceyap88** [fja05680/pinkfish#3]
Hi,
   I have found that the line 75 of the trade.py should be commented, not to default to long when exiting the trade.
   Kindly check and confirm it. Thank you.

Regards,
Vincent Yap

**Comment by fja05680** [comment]
Hi Vincent,

Sorry for taking this long to get back to you.  I was finishing my semester with finals and it got too busy for anything else.  Yes, you are correct that it should not default to long.  I actually intentionally set it to long because there are likely many other things that need to be changed to make pinkfish work with short strategies.  I've only tested long strategies.  I added some basic infrastructure so that adding short strategies to the library in the future would be easier.  I'm going to leave it as is for now until I can look at short strategies in general and what changes are required for pinkfish.  Thanks for checking out pinkfish, I really appreciate it.

Farrell

**Comment by vinceyap88** [comment]
Hi Farrell,   Thank you for your response too.Best Regards,Vincent Yap    On 11 May 2016, at 9:26 PM, Farrell Aultman <notifications@github.com> wrote:Hi Vincent,Sorry for taking this long to get back to you.  I was finishing my semester with finals and it got too busy for anything else.  Yes, you are correct that it should not default to long.  I actually intentionally set it to long because there are likely many other things that need to be changed to make pinkfish work with short strategies.  I've only tested long strategies.  I added some basic infrastructure so that adding short strategies to the library in the future would be easier.  I'm going to leave it as is for now until I can look at short strategies in general and what changes are required for pinkfish.  Thanks for checking out pinkfish, I really appreciate it.Farrell—You are receiving this because you authored the thread.Reply to this email directly or view it on GitHub
